### PR TITLE
add the two new subdomains to usegalaxy.eu

### PR DESCRIPTION
**Pull request by bgruening**
ping @mmiladi and @Florian-H-Lab

### CODE REVIEW

**Comment by bgruening** [comment]
This is for the front page, so your subsites are linked out.

File: _includes/home_carousel_subdomains.html [hunk 1]
```diff
@@ -17,12 +17,20 @@ <h2 class="display-3">Domain-specific European Galaxies</h2>
...
                 <td><a href="https://hicexplorer.usegalaxy.eu">hicexplorer.usegalaxy.eu</a></td>
                 <td>The place to go for all of your HiC analyses</td>
             </tr>
+            <tr>
+                <td><a href="https://clipseq.usegalaxy.eu">clipseq.usegalaxy.eu</a></td>
+                <td>The place to go for all of your CLIP analyses</td>
+            </tr>
+            <tr>
+                <td><a href="https://graphclust.usegalaxy.eu">graphclust.usegalaxy.eu</a></td>
+                <td>An instance focued on structural clustering of RNA secondary structures</td>
```
Comment (mmiladi) [inline _includes/home_carousel_subdomains.html]: An instance focuSed on clustering of RNAs by sequence and structure

**Comment by mmiladi** [comment]
Thanks Bjoern.

**Comment by bgruening** [comment]
Thanks @mmiladi. Online in 5min.

**Pull request merged**